editor/editor_help.cpp

```diff
@@ -577,6 +593,32 @@ void EditorHelp::_update_doc() {
 				class_desc->pop();
 			}
 
+			if (cd.is_script_doc && (cd.properties[i].setter != "" || cd.properties[i].getter != "")) {
+				class_desc->push_color(symbol_color);
+				class_desc->add_text(" [" + TTR("property:") + " ");
+				class_desc->pop(); // color
+
+				if (cd.properties[i].setter != "") {
+					class_desc->push_color(value_color);
+					class_desc->add_text("setter");
```

Comment:
Should this and `"getter"` use `TTR()`? Not sure how I'd translate them in French though :P

Reply:
Looks like existing code didn't translate those which are use somewhat like keywords, so I guess maybe it's OK as is.